### PR TITLE
Add models for learning resources

### PR DESCRIPTION
**Pull request by Vetal-Koz**
Create models

### CODE REVIEW

File: teamsite/signUp/views.py [hunk 1]
```diff
@@ -5,6 +5,10 @@
 from django.contrib.auth import login as logIn
 from django.http import HttpResponse
 from .forms import SignUpForm
+from django.contrib.auth.decorators import login_required
+# Create your views here.
```
Comment (StanioHrutc) [inline teamsite/signUp/views.py:9]: no need in this comment

File: teamsite/signUp/views.py [hunk 2]
```diff
@@ -42,4 +46,6 @@ def login(request):
     data = {
         'error': error,
     }
-    return render(request, 'signUp/login1.html', data)
\ No newline at end of file
+    return render(request, 'signUp/login1.html', data)
```
Comment (StanioHrutc) [inline teamsite/signUp/views.py:49]: really odd name for a template, please be more specific

File: teamsite/main/views.py [hunk 3]
```diff
@@ -1,5 +1,7 @@
+from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
 
 
+
```
Comment (StanioHrutc) [inline teamsite/main/views.py:5]: too many blank lines

File: teamsite/main/models.py [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = 'Course'
+        verbose_name_plural = 'Courses'
+
+
+class Themes(models.Model):
```
Comment (StanioHrutc) [inline teamsite/main/models.py:19]: Themes ? Maybe you meant Topics?

File: teamsite/main/models.py [hunk 5]
```diff
@@ -1,3 +1,29 @@
...
+    full_text = models.TextField()
+    course_image = models.ImageField(null=True, blank=True, upload_to='images/')
+    created_data = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = 'Course'
+        verbose_name_plural = 'Courses'
```
Comment (StanioHrutc) [inline teamsite/main/models.py:16]: nice!

File: teamsite/main/models.py [hunk 6]
```diff
@@ -1,3 +1,29 @@
 from django.db import models
 
-# Create your models here.
+
+class Courses(models.Model):
+    name = models.CharField(max_length=120, blank=False)
```
Comment (StanioHrutc) [inline teamsite/main/models.py:5]: It's better to use here: `title`

File: teamsite/main/models.py [hunk 7]
```diff
@@ -1,3 +1,29 @@
...
+    description = models.CharField(max_length=200, blank=False)
+    full_text = models.TextField()
+    course_image = models.ImageField(null=True, blank=True, upload_to='images/')
+    created_data = models.DateTimeField(auto_now_add=True)
```
Comment (StanioHrutc) [inline teamsite/main/models.py:9]: not created_data, but created_date, and it's better to rename to: created_at

**Comment by Vetal-Koz** [comment]
I made Django models for learning courses, and corect some code in settings.py, because it was problem with runserver. There are two models. First is about the course as a whole, second - course topics

File: teamsite/main/models.py (hunk 7 above)
Comment (Vetal-Koz) [reply teamsite/main/models.py:9]: ok

**Pull request merged**